sources/common/source_utils.go

```diff
@@ -0,0 +1,18 @@
+package common
```

Comment:
Maybe change the file from source_utils.go to utils.go?
Not sure the source is needed.

Reply:
done